File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/python/ADV-07_R.py
print("ADV-START", flush=True)
a = float(input())
b = float(input())
h = a / b
mean = (1.0 + h + 3.0) / 3.0
diff = h - h
print("OBS=MEAN:" + format(mean, ".6f") + "|DIFF:" + format(diff, ".6f"), flush=True)
print("ADV-END", flush=True)

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/kotlin/ADV-07_R.kt
fun main() {
    println("ADV-START")
    System.out.flush()
    val a: Double = readln().trim().toDouble()
    val b: Double = readln().trim().toDouble()
    val h: Double = a / b
    val mean: Double = (1.0 + h + 3.0) / 3.0
    val diff: Double = h - h
    println("OBS=MEAN:" + String.format("%.6f", mean) + "|DIFF:" + String.format("%.6f", diff))
    System.out.flush()
    println("ADV-END")
    System.out.flush()
}
